src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs

```diff
@@ -21,5 +22,15 @@ public ChairtyImportRepository(CharitiesDataContext charitiesDataContext)
 
             await tx.CommitAsync();
         }
+
+        public async Task ClearStagingData()
+        {
+            using var tx = await _charitiesDataContext.Database.BeginTransactionAsync();
+
+            await _charitiesDataContext.Database.ExecuteSqlInterpolatedAsync($"delete from CharityTrusteeStaging");
+            await _charitiesDataContext.Database.ExecuteSqlInterpolatedAsync($"delete from CharityStaging");
+
```

Comment:
would truncate be better here? I don't mind if this stays though.

Reply:
good shout. I will fix this in the next PR.